diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter6/classes/DataClassesRule.kt

```diff
@@ -144,12 +145,12 @@ class DataClassesRule(configRules: List<RulesConfig>) : DiktatRule(
     }
 
     @Suppress("UnsafeCallOnNullableType")
-    private fun ASTNode.isDataClass(): Boolean {
+    private fun ASTNode.isDataClassOrInterface(): Boolean {
```

Comment:
That's a weird method, why not use `node.psi as KtClass`, which probably already has both checks for data and interface modifiers?

Reply:
Thanks, fixed